src/libraries/System.Text.Json/src/System/Text/Json/Node/JsonNode.To.cs

```diff
@@ -14,12 +12,15 @@ public abstract partial class JsonNode
         /// <returns>JSON representation of current instance.</returns>
         public string ToJsonString(JsonSerializerOptions? options = null)
         {
-            var output = new ArrayBufferWriter<byte>();
-            using (var writer = new Utf8JsonWriter(output, options == null ? default(JsonWriterOptions) : options.GetWriterOptions()))
+            using (var output = new PooledByteBufferWriter(JsonSerializerOptions.BufferSizeDefault))
```

Comment:
Nit: the outer `using` statement can be written with the `using var` syntax to avoid one degree of nestedness.